portal-api/src/v1/users/controller.js

```diff
@@ -241,8 +241,12 @@ exports.disable = async (_id, callback) => {
 };
 
 exports.remove = async (_id, callback) => {
-  const collection = await db.getCollection('users');
-  const status = await collection.deleteOne({ _id: ObjectId(_id) });
+  if (ObjectId.isValid(_id)) {
+    const collection = await db.getCollection('users');
+    const status = await collection.deleteOne({ _id: { $eq: ObjectId(_id) } });
 
-  callback(null, status);
+    callback(null, status);
+  }
+
+  callback('Invalid portal user id', 400);
```

Comment:
We should return on or after line 248, otherwise the `callback` outside the `if` statement will run as well

Reply:
Good, spot have added.